CHANGELOG.md

```diff
@@ -271,717 +271,15 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
-   - Default sorting changed
-   - Default label pattern changed from `[auth][year]` to `[authors3][year]`
- - Feature: case changers now leave protected areas (enclosed with curly brackets) alone
- - BREAKING: The BibTeX key generator settings from previous versions are lost
- - BREAKING: LabelPatterns `[auth.etal]`, `[authEtAl]`, `[authors]`, `[authorsN]`, `[authorLast]` and more to omit spaces and commas (and work as described at http://jabref.sourceforge.net/help/LabelPatterns.php)
- - BREAKING: `[keywordN]` returns the Nth keyword (as described in the help) and not the first N keywords
- - BREAKING: If field consists of blanks only or an emtpy string, it is not written at all
- - Feature: new LabelPattern `[authFirstFull]` returning the last name of the first author and also a "van" or "von" if it exists
- - Feature: all new lines when writing an entry are obeying the globally configured new line (File -> newline separator). Affects fields: abstract and review
- - Feature: `[veryShortTitle]` and `[shortTitle]` also skip words like "in", "among", "before", ...
- - Feature: New LabelPattern `[keywordsN]`, where N is optional. Returns the first N keywords. If no N is specified ("`[keywords]`"), all keywords are returned. Spaces are removed.
- - Update supported LookAndFeels
- - Show replaced journal abbreviations on console
- - Integrated [GVK-Plugin](http://www.gbv.de/wikis/cls/Jabref-GVK-Plugin)
- - The three options to manage file references are moved to their own separated group in the Tools menu.
- - Default preferences: Remote server (port 6050) always started on first JabRef instance. This prevents JabRef loaded twice when opening a bib file.
-
-### Fixed
- - Fixed the bug that the file encoding was not correctly determined from the first (or second) line
- - Fixed [#325](https://github.com/JabRef/jabref/issues/325): Deactivating AutoCompletion crashes EntryEditor
- - Fixed bug when having added and then removed a personal journal list, an exception is always shown on startup
- - Fixed a bug in the IEEEXploreFetcher
- - Fixed [bug 1282](https://sourceforge.net/p/jabref/bugs/1282/) related to backslashes duplication.
- - Fixed [bug 1285](https://sourceforge.net/p/jabref/bugs/1285/): Editing position is not lost on saving
- - Fixed [bug 1297](https://sourceforge.net/p/jabref/bugs/1297/): No console message on closing
- - Fixed [#194](https://github.com/JabRef/jabref/issues/194): JabRef starts again on Win XP and Win Vista
- - Fixed: Tooltips are now shown for the #-field when the bibtex entry is incomplete.
- - Fixed [#173](https://github.com/JabRef/jabref/issues/173): Personal journal abbreviation list is not loaded twice
- - Bugfix: Preview of external journal abbreviation list now displays the correct list
- - Fixed [#223](https://github.com/JabRef/jabref/issues/223): Window is displayed in visible area even when having multiple screens
```

Comment:
We need such a text for 3.x too

Reply:
Which text? 😇 

The 3.0 changelog points back to the 2.11 one. But not forward to 4.0 though...